src/Sentry.Maui/SentryMauiOptions.cs

```diff
@@ -65,4 +65,35 @@ public SentryMauiOptions()
     /// Screenshots can be removed from some specific events during BeforeSend through the Hint.
     /// </remarks>
     public bool AttachScreenshot { get; set; }
+    private Func<SentryEvent, SentryHint, SentryEvent?>? _beforeSend;
+    /// <summary>
+    /// Action performed before attaching a screenshot
+    /// </summary>
+    internal Func<SentryEvent, SentryHint, SentryEvent?>? BeforeCaptureScreenshotInternal => _beforeSend;
```

Comment:
I got a little bit confused that the internal member is called `_beforeSend` but the public members that set it are called `BeforeCaptureScreenshot`. I'd probably rename the internal member `_beforeCaptureScreenshot` and it could probably be made private.

Reply:
I think you are right, let's rename it